Jellyfin.Server/Program.cs

```diff
@@ -221,6 +223,14 @@ private static async Task StartApp(StartupOptions options)
             finally
             {
                 appHost.Dispose();
+                _logger.LogInformation("Running query planner optimizations in the database... This might take a while");
+
+                // Run after disposing the application
+                using var context = new JellyfinDbProvider(appHost.ServiceProvider, appPaths).CreateContext();
```

Comment:
You're accessing a property on a disposed object

Reply:
@cvium Oh true. But how is possible this works then? This means we're not disposing it properly as it's still available?

I guess switching the order and running this before dispose is enough anyway.

Reply:
Being disposed doesn't mean it's completely gone. It has to be picked up by the GC.